maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -128,15 +132,21 @@ def get_header(self, step):
             if resources[key]:
                 modified_header.append(value.format(**resources))
 
-        if "procs" in self._batch:
-            modified_header.append(
-                "#SBATCH --ntasks={}".format(resources["procs"])
-            )
-
         exclusive = resources.get("exclusive", False)
         if exclusive:
             modified_header.append("#SBATCH --exclusive")
 
+        # If neither Procs nor Nodes exist, throw an error
+        procs = resources.get("procs")
+        nodes = resources.get("nodes")
+        if not procs and not nodes:
+            err_msg = "At least one of either (procs) or (nodes) in {} needs "
+            "to be present for Script to proceed.".format(step.name)
+            LOGGER.error(err_msg)
+            rt_err_msg = "No explicit resources specified in {}. At least one of " 
+            "(nodes) or (procs) must be set to a non-zero value.".format(step.name)
```

Comment:
Sorry, I entirely mis-typed. When I said parentheses I totally meant quotations. Sorry about that. You can also use the same error message for both the logging and exception.